### ISSUE TITLE
False positive for oelint.vars.multilineident

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#49]
Snippet:

```
UPSTREAM_CHECK_REGEX = "(?P<pver>12\.\d+\.\d+)"
```
